mongodb-advanced/components/people-service.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+export default function factory($http, brAlertService) {
+  const service = {};
+
+  service.savePerson = (person, callback) => {
+    return $http.post('/people/', person)
+      .then(response => {
+        callback();
+      })
+      .catch(err => {
```

Comment:
because the response needs to be passed through to the controller, the `catch` typically occurs in the controller, not in the service.  So, add this same catch code to the end of the promise chain in the controller.